.github/workflows/sync_project.yml

```
@@ -0,0 +1,32 @@
+name: Sync qube project
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+        - name: Setup Python
+          uses: actions/setup-python@v1
+          with:
+            python-version: 3.8
+
+        - name: Install qube
+          run: pip install qube
+
+        - uses: actions/checkout@v2
+          with:
+              fetch-depth: 0
```

```suggestion
              fetch-depth: 0
              token: '${{ secrets.JOHNNY_Q5_REPORTS_TOKEN}}'
```
this was added by @sven1103 to my project, it probably means something

---

Good Catch